internal/store/organizations.go

```diff
@@ -55,11 +57,12 @@ func (s *Store) Organizations(opts *atlas.OrganizationsListOptions) (*atlas.Orga
 }
 
 // Organization encapsulate the logic to manage different cloud providers.
-func (s *Store) Organization(id string) (interface{}, error) {
+func (s *Store) Organization(id string) (*atlas.Organization, error) {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
 		result, _, err := s.clientv2.OrganizationsApi.GetOrganization(s.ctx, id).Execute()
-		return result, err
+		newOrg := atlas.Organization{ID: *result.Id, IsDeleted: result.IsDeleted, Name: result.Name, Links: mapLinks(result.Links)}
```

Comment:
[q] what happens to the result if the endpoint throws an error? (e.g. 404)  might we get some panic error trying to create the org object here?

Reply:
Good point, thanks!
Updated with some error/nil handling (and moved to a separate function)